corehq/apps/app_manager/models.py

```diff
@@ -987,6 +987,7 @@ def wrapped_xform(self):
     def validate_form(self):
         vc = self.validation_cache
         if vc is None:
+            # todo: now that we don't use formtranslate, does this still apply?
```

Comment:
This was added because hashtags in the formbuilder write xml binds/setvalues like:

`<bind nodeset="/data/some/quetsion"  calculate="/data/other/question + 1" vellum:calculate="#form/other/question + 1" />`

App validation failed whenever it ran into `vellum:calculate` because #form isn't a valid xpath. it wasn't clear why it cares that it isn't a valid xpath since it would be ignored by the phone. I don't know if that was fixed by formplayer's validation though